### PR TITLE
Forward port #25 and #26 to 10.0-dev

### PR DESCRIPTION
**Pull request by isoboroff**
This merge takes #25 and #26 from the master branch and forward ports them here.

### CODE REVIEW

**Comment by isoboroff** [comment]
@cmacdonald this isn't building on Windows, can you take a look?

File: build.bat [hunk 1]
```diff
@@ -0,0 +1 @@
+cl /DVERSIONID=\"9.0.7\"  trec_eval.c formats.c meas_init.c meas_acc.c meas_avg.c meas_print_single.c meas_print_final.c get_qrels.c get_trec_results.c get_prefs.c get_qrels_prefs.c get_qrels_jg.c form_res_rels.c form_res_rels_jg.c form_prefs_counts.c utility_pool.c get_zscores.c convert_zscores.c measures.c  m_map.c m_P.c m_num_q.c m_num_ret.c m_num_rel.c m_num_rel_ret.c m_gm_map.c m_Rprec.c m_recip_rank.c m_bpref.c m_iprec_at_recall.c m_recall.c m_Rprec_mult.c m_utility.c m_11pt_avg.c m_ndcg.c m_ndcg_cut.c m_Rndcg.c m_ndcg_rel.c m_binG.c m_G.c m_rel_P.c m_success.c m_infap.c m_map_cut.c m_gm_bpref.c m_runid.c m_relstring.c m_set_P.c m_set_recall.c m_set_rel_P.c m_set_map.c m_set_F.c m_num_nonrel_judged_ret.c m_prefs_num_prefs_poss.c m_prefs_num_prefs_ful.c m_prefs_num_prefs_ful_ret.c m_prefs_simp.c m_prefs_pair.c m_prefs_avgjg.c m_prefs_avgjg_Rnonrel.c m_prefs_simp_ret.c m_prefs_pair_ret.c m_prefs_avgjg_ret.c m_prefs_avgjg_Rnonrel_ret.c m_prefs_simp_imp.c m_prefs_pair_imp.c m_prefs_avgjg_imp.c m_map_avgjg.c m_Rprec_mult_avgjg.c m_P_avgjg.c m_yaap.c windows/mman.c windows/ya_getopt.c  /link /out:trec_eval.exe
```
Comment (cmacdonald) [inline build.bat:1]: Amend version for 10

**Review by cmacdonald — COMMENTED** [review]
trec_eval-9.0.7.tar.gz Shouldn’t be in patch.

Still looking for linking problem

**Comment by cmacdonald** [comment]
"measures.obj : error LNK2001: unresolved external symbol te_meas_rbp" etc
@
https://github.com/usnistgov/trec_eval/runs/1069443260#step:3:188

I think you just have to update build.bat with the new .c files - the RBP measure isn't listed.

Background: I don't think MSVC tools provides a make, so i replicated the makefile using batch files.

**Pull request merged**